bin/testEnvLocalK8S.sh

```diff
@@ -114,6 +115,40 @@ function startLocalApiserver() {
     echo "Started local etcd and apiserver !"
 }
 
+function startIstio() {
+    ensureLocalApiServer
+    startPilot
+    startEnvoy
+    startMixer
+}
+
+function stopIstio() {
+  if [[ -f $LOG_DIR/pilot.pid ]] ; then
+    kill -9 $(cat $LOG_DIR/pilot.pid)
+    kill -9 $(cat $LOG_DIR/mixer.pid)
+    kill -9 $(cat $LOG_DIR/envoy4.pid)
+    rm $LOG_DIR/{pilot,mixer,envoy4}.pid
+  fi
+}
+
+function startPilot() {
+  POD_NAME=pilot POD_NAMESPACE=istio-system ${ISTIO_OUT}/pilot-discovery discovery \
+    -n default --kubeconfig .circleci/config &
+  echo $! > $LOG_DIR/pilot.pid
+}
+
+function startMixer() {
+  ${ISTIO_OUT}/mixs server erver --configStoreURL=fs:${ISTIO_GO}/mixer/testdata/configroot \
```

Comment:
was this ever exercised ??

Reply:
What do you mean ? Mixer does start - and will be used in new local tests for v2.
It is not enabled by default ( need to call startIstio )

Reply:
I doubt `mixs server erver ...` works